packages/core/src/routes/sign-in-experience.ts

```diff
@@ -30,12 +30,19 @@ export default function signInExperiencesRoutes<T extends AuthedRouter>(router:
   router.patch(
     '/sign-in-exp',
     koaGuard({
-      body: SignInExperiences.createGuard.omit({ id: true }).partial(),
+      body: SignInExperiences.createGuard
+        .omit({ id: true, termsOfUseUrl: true })
+        .merge(
+          object({
+            termsOfUseUrl: string().url().optional().nullable().or(literal('')),
```

Comment:
what will we store in DB is an empty string provided?

Reply:
We will convert empty strings into a `null` value in our DB.